eco/eco.go

```diff
@@ -80,7 +84,7 @@ type Fetchable interface {
 // Note that the ith element of the datafiles slice is
 // the ith factor from eco.Factors
 //
-func CalcBenefitsWithData(
+func CalcBenefitSummaryWithData(
```

Comment:
`CalcBenefitSummaryWithData` -> `CalcSummarizedBenefitsWithData` would be more consistent with the two other functions it is similar to.